crates/ruff_linter/src/rules/flake8_pytest_style/rules/fixture.rs

```diff
@@ -34,9 +34,6 @@ use super::helpers::{
 /// Either removing those unnecessary parentheses _or_ requiring them for all
 /// fixtures is fine, but it's best to be consistent.
```

Comment:
It might be good to keep a mention of what the default is (just remove the mention of preview mode):

```suggestion
/// fixtures is fine, but it's best to be consistent. The rule defaults to
/// removing unnecessary parentheses, to match the documentation of the
/// official pytest projects.
```